www/src/pages/en/usage/trpc.md

```diff
@@ -230,7 +230,7 @@ Compare this to the tRPC example above and you can see some of the advantages of
 
 Here are some snippets that might come in handy.
```

Comment:
```suggestion
Her er noen kodesnutter som kan hjelpe deg.
```

Reply:
Is this file a part of the translation? since it's under en/usage

Reply:
No, that's an error.